Documentation/compatibility/workflow-designer-accessibility.md

```diff
@@ -0,0 +1,123 @@
+## Accessibility improvements in WF workflow designer
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The Windows Workflow Foundation (WF) workflow designer is improving how it works with accessibility technologies. These improvements include the following changes:
+- Tab order changed to left->right & top->bottom in some controls
```

Comment:
Tab order changed... --> The tab order is changed to left to right and to top to bottom in some controls:

Also, each sentence should end in a period.